inference-engine/tests/ngraph_functions/include/ngraph_functions/builders.hpp

```diff
@@ -439,5 +439,12 @@ std::shared_ptr<ngraph::Node> makeNormalizeL2(const ngraph::Output<Node>& data,
                                               const std::vector<int64_t>& axes,
                                               float eps,
                                               ngraph::op::EpsMode epsMode);
+
+std::shared_ptr<Node> makeGatherND(const ngraph::Output<Node>& input,
```

Comment:
Is it a duplicate of the function on line #429?

Reply:
good catch. removed.